pkg/sql/sem/builtins/geo_builtins_test.go

```diff
@@ -103,3 +104,88 @@ func TestGeoBuiltinsPointEmptyArgs(t *testing.T) {
 		})
 	}
 }
+
+func mustParseDFloat(t testing.TB, in string) tree.Datum {
```

Comment:
please remove these tests and add it as a logic test to https://github.com/cockroachdb/cockroach/blob/master/pkg/sql/logictest/testdata/logic_test/geospatial instead. use `make testbaselogic FILES='geospatial'  TESTFLAGS='-rewrite'` to rewrite the output and verify it is correct